benchmarks/benchmark/tools/locust-load-inference/locust-docker/locust-tasks/run.sh

```diff
@@ -35,7 +37,10 @@ if [[ "$LOCUST_MODE" = "master" ]]; then
         LOCUST_OPTS="$LOCUST_OPTS --stop-timeout $STOP_TIMEOUT"
     fi
 elif [[ "$LOCUST_MODE" = "worker" ]]; then
+
+    echo "login to huggingface"
     huggingface-cli login --token $HUGGINGFACE_TOKEN
```

Comment:
Does the login have any existing output? Is it possible to log something helpful on login failure?

Reply:
This is the output it gives around the existing comments:

"Token is valid (permission: read).
Your token has been saved to /root/.cache/huggingface/token"

I wanted additional logging as my previous issue wasn't showing any logs (even the login ones) so the echo commands were to identify what wasn't running.